drivers/storage/scaleio/tests/test-env-up.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This script launches infrastructure required for testing the ScaleIO storage driver.
+# It spins up VPC and EC2 instance so AWS account owner will get charged for time
+# that resources will be running.
+
+set -e
+# set -x
+
+AWS_ACCESS_KEY="$1"
+AWS_SECRET_KEY="$2"
+CF_STACK_NAME="$3"
+LAUNCH_KEY_NAME="$4"
+
+# Make sure that aws cli is installed
+hash aws 2>/dev/null || {
```

Comment:
ok. That section was copied from the EFS tests. I can include the installation of the tools as well.